src/products/android-l10n/mobile_l10n_faqs.md

```diff
@@ -0,0 +1,139 @@
+
+One reason to use hardcoded brand names is when you always want the text to mention that specific brand name, e.g. Mozilla or Firefox. One example is telemetry, as shown in this [issue](https://github.com/mozilla-l10n/focusios-l10n/pull/119#discussion_r1919628152): if telemetry is enabled, the data is always sent to Mozilla, not to a fork’s third-part vendor.
+
+### When do I have to use a different ID when updating existing strings?
+
+[This document](https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html#general-guidelines) explains in details why and when it is necessary to use new string IDs when updating existing strings. As a general guideline:
+
+* If you are changing a string such that its **meaning has changed**, you must update the string ID. That will force retranslation in Pontoon across all locales.
+* If your changes are relevant only for English — for example, to correct a typographical error or to make capitalization consistent — then there is generally no need to update the string ID.
+
+### Plural support
+
+We do not handle [proper plural support](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#use-proper-plural-forms) for mobile yet. For this reason, you should try to use plural neuter expressions, where there isn’t a noun associated to the number: for example, instead of `%S files were copied`, use `Copied files: %S`.
+
+If in doubt reach out to the current localization mobile ePM or on the Slack #l10n channel.
+
+### Use localization notes and avoid using comments as development notes
+
+Localizers typically see one string at a time, without access to the full source code or interface. As a result, they may struggle to understand the meaning or context of a string.
+
+Notes targeting developers can be unclear or overly technical — rather than giving context which would be most helpful for translation. To support accurate and meaningful translations, it’s essential to include clear, concise, and context-rich comments specifically written for localizers.
+
+These comments are generally referred to as *localization notes*. Please read more about this in our [best practices guide](https://mozilla-l10n.github.io/documentation/localization/dev_best_practices.html#add-localization-notes%20).
+
+## Translation completion and community
+
+### [Is my feature going to be localized in language X before launch?](../firefox_desktop/firefox_l10n_faqs.md#is-my-feature-going-to-be-localized-in-language-x-before-launch)
+
+### I see pending suggestions in Pontoon, how can I get them approved?
+
+Reach out to the current localization mobile ePM or on the Slack #l10n channel.
```

Comment:
Re-reading, this should be integrated (see the desktop FAQ). It's not just about asking you to approve them.